hoomd/md/pair/aniso.py

```diff
@@ -247,3 +250,248 @@ def type_shapes(self):
+        alj = hoomd.md.pair.aniso.ALJ(nl, r_cut=2.5)
+
+        cube_verts = [(-0.5, -0.5, -0.5),
+                      (-0.5, -0.5, 0.5),
+                      (-0.5, 0.5, -0.5),
+                      (-0.5, 0.5, 0.5),
+                      (0.5, -0.5, -0.5),
+                      (0.5, -0.5, 0.5),
+                      (0.5, 0.5, -0.5),
+                      (0.5, 0.5, 0.5)];
+
+        cube_faces = [[0, 2, 6],
+                      [6, 4, 0],
+                      [5, 0, 4],
+                      [5,1,0],
+                      [5,4,6],
+                      [5,6,7],
+                      [3,2,0],
+                      [3,0,1],
+                      [3,6,2],
+                      [3,7,6],
+                      [3,1,5],
+                      [3,5,7]]
+
+        alj.params[("A", "A")] = dict(epsilon=2.0,
+                                      sigma_i=1.0,
+                                      sigma_j=1.0,
+                                      alpha=1,
+                                      )
+        alj.shape["A"] = dict(vertices=cube_verts,
+                              faces=cube_faces,
+                              rounding_radii=1)
+
+    Warning:
+        Changing dimension in a simulation will invalidate this force and will
+        lead to error or unrealistic behavior.
+    """
+
+    # We don't define a _cpp_class_name since the dimension is a template
+    # parameter in C++, so use an instance level attribute instead that is
+    # created in _attach based on the dimension of the associated simulation.
+
+    def __init__(self, nlist, default_r_cut=None, mode='none'):
+        super().__init__(nlist, default_r_cut, mode)
+        params = TypeParameter(
+            'params', 'particle_types',
+            TypeParameterDict(epsilon=float,
+                              sigma_i=float,
+                              sigma_j=float,
+                              alpha=int,
+                              contact_ratio_i=0.15,
+                              contact_ratio_j=0.15,
+                              average_simplices=True,
+                              len_keys=2))
+
+        shape = TypeParameter(
+            'shape', 'particle_types',
+            TypeParameterDict(vertices=[(float, float, float)],
+                              faces=[[int]],
+                              rounding_radii=OnlyIf(
+                                  to_type_converter((float, float, float)),
+                                  preprocess=self._to_three_tuple),
+                              len_keys=1))
+
+        self._extend_typeparam((params, shape))
+
+    @staticmethod
+    def get_ordered_vertices(vertices, return_faces=True):
+        """Compute vertices and faces of a convex hull of given vertices.
+
+        Warning:
+            This method requires the ``coxeter`` package which is available on
+            PyPi and conda-forge by that name, and can be found on GitHub under
+            ``glotzerlab/coxeter``.
```

Comment:
```suggestion
            This method requires the `coxeter <https://coxeter.readthedocs.io/>`_ package.
```
The coxeter documentation provides the single source of truth.